course6/toyapp_thomasdhuys/drawlib/matrix.h

```diff
@@ -0,0 +1,38 @@
+#pragma once
+
+#include <array>
+
+#include "vertex.h"
+
+namespace Draw {
+
+namespace {
+	static constexpr size_t COLS = 4;
+	static constexpr size_t ROWS = 4;
+}
+
+using RowType = std::array<VCoord, COLS>;
+
+class Matrix4x4 {
+private:
+	std::array< RowType, ROWS> data;
```

Comment:
If the class itself contains "4x4" in it's name, what's the value of making ROWS and COLS defined constants? Just put 4 in there, and save yourself some typing, and your readers some reading.